Neos.Fusion/Classes/Core/Cache/ParserCache.php

```diff
@@ -105,4 +108,29 @@ private function getAbsolutePathForPackageRessourceUri(string $requestedPath): s
         $package = $this->packageManager->getPackage($resourceUriParts['host']);
         return Files::concatenatePaths([$package->getResourcesPath(), $resourceUriParts['path']]);
     }
+
+    /**
+     * Uses the same technique to resolve a package resource URI like Flow.
+     *
+     * package://My.Site/NodeTypes/Foo/Bar.fusion
+     * ->
+     * FLOW_PATH_ROOT/Packages/Sites/My.Package/NodeTypes/Foo/Bar.fusion
+     *
+     * {@see \Neos\Flow\ResourceManagement\Streams\ResourceStreamWrapper::evaluateResourcePath()}
+     * {@link https://github.com/neos/flow-development-collection/issues/2687}
+     *
+     * @throws \InvalidArgumentException
+     */
+    private function getAbsolutePathForNodetypesUri(string $requestedPath): string
+    {
+        $packageUriParts = UnicodeFunctions::parse_url($requestedPath);
+
+        if ((isset($packageUriParts['scheme']) === false
+            || $packageUriParts['scheme'] !== 'package')) {
```

Comment:
should be nodetypes ;)

Reply:
Did not manage to adjust this yet. Will remove all the automatic soon

Reply:
okay bo problem i just wanted to make sure that we are on the same side and this doesnt get merged as is ^^